python-sdk/src/astro/sql/table.py

```diff
@@ -112,3 +117,19 @@ def name(self, value: str) -> None:
         if not isinstance(value, property) and value != self._name:
             self._name = value
             self.temp = False
+
+    @uri.default
+    def _path_to_dataset_uri(self) -> str:
+        """Build a URI to be passed to Dataset obj introduced in Airflow 2.4"""
+        from urllib.parse import urlencode, urlparse
+
+        path = f"astro://{self.conn_id}@"
```

Comment:
```suggestion
        path = f"astro+Table://{self.conn_id}@"
```

Reply:
@kaxil Would it be a good idea to add a Table or File in the URL? would be more explicit and will give a nice namespace to Table and File objects. Also, prevent any collisions among File URI and Table URI or any other data placeholder we may add.

Reply:
If we agree we can do this for Files as well.

Reply:
That doesn't give any new info, "scheme" for files will be `astro+gs` or `astro+s3` so just change it to files doesn't make sense in my opinion

Reply:
@kaxil I believe that we should not add "Table" in the URI prefix since it does not give much information and is not consistent with how we intend to represent the `File` URI prefix (`astro+gs` or `astro+s3`, as you described).

It feels it would be more symmetric if we added to the URI prefix the prefix used to access the dataset in question. For databases, this could be:
* PostgresSQL: astro+[postgresql](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING)
* SQLite: "astro+[file](https://www.sqlite.org/uri.html)"

Another alternative could be to use only `astro` as the URI prefix - since the remaining information is part of `conn_id`.

I also believe we should update the docs with the logic used to represent datasets.

Reply:
@kaxil I'm not suggesting a replacement of `gs/s3` instead `file+file+gs` or `file+file+s3` that was with string we can know that it's coming from a file object or table. Just a suggestion to be a little verbose.

Reply:
But @utkarsharma2 doesn't `gs` or `s3` suggest the file itself?

Reply:
That starts looking too verbose in the UI once you have many datasets:

<img width="703" alt="image" src="https://user-images.githubusercontent.com/8811558/189127020-b18ff4cf-cc57-4e93-8a5d-1a1aabdc6275.png">

Reply:
oh, I didn't think of that.